### DISCUSSION TITLE
vcpkg on visual studio 2010

### DISCUSSION THREAD

**Discussion by vinben007** [microsoft/vcpkg#20683] [Q&A]
Hello everyone,

I'd appreciate it if you let me know if we can "integrate" vcpkg on visual studio 2010. I need to make it work on this version of visual studio.

Thank you very much for your help.

regards,

Alexandre BENOIT

**Comment by autoantwort** [comment]
You can install libs with vcpkg and then manually set the library include and library search paths like for every other lib.
